src/index.js

```diff
@@ -28,7 +29,8 @@ const self = module.exports = {
           },
           port: config.browsersync.port || 3000,
           notify: config.browsersync.notify,
-          tunnel: config.browsersync.tunnel
+          tunnel: config.browsersync.tunnel,
+          open
```

Comment:
Can we please have `open: config.browsersync.open || true,` instead?

Reply:
A valid option is `open: false`, which won't work that way. Object destructuring, however, allows for this.

Reply:
Ooops, you're right 😀